app/controllers/errors_controller.rb

```diff
@@ -5,8 +5,8 @@ def not_found
     render status: :not_found
   end
 
-  def unprocessable_entity
-    render status: :unprocessable_entity
+  def unprocessable_content
```

Comment:
So, I had an outstanding question about what this controller action is doing for us. Is the method ever getting used?

Reply:
After looking into this it doesn't seem to be used (since the removal in the routes file)